### ISSUE TITLE
Support Tuya 0d devices 

### ISSUE THREAD

**Issue by volker76** [volker76/tuya_mqtt.net#6]
see this discussion [github.com/codetheweb/tuyapi](https://github.com/codetheweb/tuyapi/issues/389#issuecomment-760803032)

there are devices which cannot be read with the DP_QUERY

requires code like
```
var jsonIn = dev.FillJson("{\"dps\":{\"1\":null,\"2\":null,\"100\":null,\"102\":null,\"103\":null}}", true,true,true,true);
byte[] request = dev.EncodeRequest(TuyaCommand.CONTROL_NEW, jsonIn);
```

**Comment by volker76** [comment]
in progress - see branch device-suport-od

**Comment by volker76** [comment]
adressed in https://github.com/volker76/tuya_mqtt.net/commit/498c74bfc493a32cee9d1b2e08a647828647cd0f

**Comment by volker76** [comment]
closed with version v1.0.0
